packer/install-dependencies.sh

```diff
@@ -6,20 +6,52 @@ echo "Starting build on " `date` > /var/log/azure/golden-image-install.complete
 echo "Using kernel:" >> /var/log/azure/golden-image-install.complete
 cat /proc/version | tee -a /var/log/azure/golden-image-install.complete
 
+echo ""
+echo "Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):" >> /var/log/azure/golden-image-install.complete
+
 ETCD_VERSION="3.2.25"
 ETCD_DOWNLOAD_URL="https://acs-mirror.azureedge.net/github-coreos"
 installEtcd
+echo "  - etcd v${ETCD_VERSION}" >> /var/log/azure/golden-image-install.complete
 
 installDeps
+cat << EOF >> /var/log/azure/golden-image-install.complete
```

Comment:
Can we move this to a variable  - `/var/log/azure/golden-image-install.complete`?

Reply:
The reason for this being a file is that it is written to disk so that every aks-engine vm has that file for debugging

Reply:
Oh, I meant assign the file name to a variable.

Reply:
ohh gotcha, sure, I can that